GAE/src/org/akvo/flow/domain/persistent/SurveyAssignment.java

```diff
@@ -25,21 +25,29 @@
 import com.gallatinsystems.framework.domain.BaseDomain;
 
 /**
- * domain to store assignment of surveys to devices
- * 
- * @author Christopher Fagiani
+ * domain to store assignment of forms to devices
  */
+//TODO: could be renamed while referencing entities like this: @PersistenceCapable(table="SurveyAssignment")
```

Comment:
Is this done?

Reply:
I did for a while but realised that SurveyAssignment was actually a reasonable name now.
I'd love to do Survey->Form, but perhaps after the SQL refactring...

Reply:
Removed the comment